### PR TITLE
Add newsletter widget to news archive hero

### PR DESCRIPTION
**Pull request by sebastian-ew**
This PR creates a new template to be added in the news archive page. Some changes are still pending to be done directly in the stage environment.

### CODE REVIEW

File: fw-child/languages/cdc/fr_CA.po [hunk 1]
```diff
@@ -1353,3 +1354,11 @@ msgstr "Sélectionnez pour afficher soit les valeurs de données absolues, soit
 #: template/map/controls/display.php:46
 msgid "View climate data as grid boxes, or aggregated by Census Subdivision, Health Region or Watershed."
 msgstr "Affichez les données climatiques sous forme de grilles ou regroupées par subdivision de recensement, région socio-sanitaire ou bassin versant."
+
+#: template/newsletter-hero.php:3
+msgid "Climatedata.ca Newsletter"
```
Comment (xfrenette-crim) [inline fw-child/languages/cdc/fr_CA.po:1359]: Official website name is "ClimateData.ca" (with uppercase "D")

**Review by xfrenette-crim — CHANGES_REQUESTED** [review]
The new file `fw-child/template/newsletter-hero.php` is almost an exact copy of the existing file `fw-child/template/newsletter-signup.php`. Instead of recreating the MailChimp form in a new file, I would reuse the existing file (easier to maintain if the MailChimp form is in a single file). I would then create the hero's "newsletter" box using the Builder (with all texts defined in the builder and not in the PHP).

You probably already saw that the CSS code (through Bootstrap) defines a lot a "basic" classes (for padding, for margin, for borders, etc.). When we need to create new styles, I suggest we continue in this vein (if applicable). So, for example, instead of creating a class just for the newsletter box, I would create a new generic "bg-blur" class that can then be used in the Builder (alongside other classes) to create the style of the newsletter box.

I pushed in this commit (a4185fb) the suggestions I describe above.

Then, in the builder, I create a new column for the newsletter box in the hero and apply it the required classes (`blurred-bg p-3 mt-5 mt-xl-0 border border-gray-600`, may need to be adjusted for responsive).

Inside this column, I insert a text block and set the correct styles on the paragraphs:
```html
<p class="all-caps text-blue-100 mb-2">ClimateData.ca Newsletter</p>
<p>Join our newsletter for periodic updates.</p>
```
(A bottom margin `mb-3` was added to the block.)

Then, still in the same column, I insert the template `fw-child/template/newsletter-signup.php`.

On the surrounding row block, we can add the class `align-items-start` to put a `align-items: flex-start` to ensure a "compact" height for the newsletter box.

## Note about the name

When writing the website's address, we use "ClimateData.ca" and "DonneesClimatiques.ca" (with CamelCase, no accent)

File: fw-child/languages/cdc/fr_CA.po [hunk 2]
```diff
@@ -1353,3 +1354,11 @@ msgstr "Sélectionnez pour afficher soit les valeurs de données absolues, soit
...
+msgstr "Donneesclimatiques.ca Infolettre"
```
[the remark this answers is not in the document]
Comment (MahieddineAk) [reply fw-child/languages/cdc/fr_CA.po]: No need for this as the `newsletter-hero.php` file has been disregarded.

File: fw-child/languages/cdc/fr_CA.pot [hunk 3]
```diff
@@ -1292,14 +1292,23 @@ msgstr ""
 msgid "All Variables"
 msgstr ""
 
-#: template/newsletter-signup.php:3
-msgid "Join the newsletter"
+#: template/newsletter-hero.php:3
+msgid "Climatedata.ca Newsletter"
```
[the remark this answers is not in the document]
Comment (MahieddineAk) [reply fw-child/languages/cdc/fr_CA.pot]: No need for this as the `newsletter-hero.php` file has been disregarded.

File: fw-child/languages/cdc/fr_CA.po (hunk 1 above)
Comment (MahieddineAk) [reply fw-child/languages/cdc/fr_CA.po]: No need for this as the `newsletter-hero.php` file has been disregarded.

**Comment by MahieddineAk** [comment]
@xfrenette-crim I see your point regarding relying more on the Builder rather than custom coding for the newsletter. I have adopted your suggestion (cherry-pick changes), and I will merge this PR to proceed with the Builder changes.

**Pull request merged**